api/application.go

```diff
@@ -358,6 +358,30 @@ func (a *ApplicationAPI) UploadApplicationImage(ctx *gin.Context) {
 	})
 }
 
```

Comment:
> I don't know if the swagger comments are generated or if I have to manually add them before the function.

You need to create the swagger comment manually above the method and then run `make download-tools update-swagger` afterward.

Reply:
Will do.